packages/client/package.json

```diff
@@ -8,6 +8,7 @@
     "wait:client-api": "wait-on ../client-api/dist/index.d.ts"
   },
   "dependencies": {
+    "morphdom": "^2.7.4",
```

Comment:
set an exact version and then `pnpm i` to add it to the lock file 👍

Reply:
Good catch, thanks!